discovery-infra/bootstrap_in_place.py

```diff
@@ -38,10 +40,16 @@ def installer_generate(openshift_release_image):
 
 @utils.retry(exceptions=Exception, tries=5, delay=30)
 def installer_gather(ip, ssh_key, out_dir):
-    _stdout, stderr, _ret = utils.run_command(
-        f"{INSTALLER_BINARY} gather bootstrap --bootstrap {ip} --master {ip} --key {ssh_key}"
+    stdout, stderr, _ret = utils.run_command(
```

Comment:
Consider refactoring utils.run_command to get optional stdout and stderr.

Reply:
Not sure what you mean. It's already returning stdout and stderr?